openshift/log-detective.yaml

```diff
@@ -50,8 +50,8 @@ spec:
     - ReadWriteOnce
   resources:
     requests:
-      storage: 100Mi
-status: {}
+      storage: 2Gi
+  storageClassName: gp2
```

Comment:
Why not gp3?

Reply:
updated & dropped the line

The owners of communishift set this up, we should ask them if gp3 is available.